packages/plugin/src/protobufts-plugin.ts

```diff
@@ -28,7 +28,7 @@ import {ServiceClientGeneratorGrpc} from "./code-gen/service-client-generator-gr
 import * as ts from "typescript";
 
 
-export class ProtobuftsPlugin extends PluginBase {
+export class ProtobuftsPlugin extends PluginBase<GeneratedFile> {
```

Comment:
not sure what was wrong , but TS 3.9.10 didn't want to compile this

Reply:
`PluginBase` got a default type recently: https://github.com/timostamm/protobuf-ts/commit/4304289f4ed83f60d9ba5d6b1b55abc48db84d8d#diff-89c13abbe6b7815deac020cfd77b59a89c17f6773cc969d5ff9bce6c4830d0f1 

Can you try `cd packages/plugin-framework && make build`? That should fix it.

Reply:
ah, right 🤦 
pushed the update